src/NuGet.ProjectModel/PackageSpecResolver.cs

```diff
@@ -57,7 +57,7 @@ public bool TryResolvePackageSpec(string name, out PackageSpec project)
 
         private void Initialize(string projectPath, string rootPath)
         {
-            _searchPaths.Add(Path.GetDirectoryName(projectPath));
+            _searchPaths.Add(new DirectoryInfo(projectPath).Parent.FullName);
 
```

Comment:
Fixing the same bug in dnx: https://github.com/aspnet/dnx/commit/7bd3b77a054cc4b43eec2786c1038d535e266066#diff-ec765d4f51c09a4149926c21f59d1568